device/barefoot/x86_64-accton_as9516bf_32d-r0/newport/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,83 @@
+{% set default_cable = '5m' %}
+{% set ingress_lossless_pool_size =  '4194304' %}
+{% set ingress_lossy_pool_size =  '7340032' %}
+{% set egress_lossless_pool_size =  '16777152' %}
+{% set egress_lossy_pool_size =  '7340032' %}
+
+{%- macro generate_port_lists(PORT_ALL) %}
+    {# Generate list of ports #}
+    {%- for port_idx in range(0,32) %}
+        {%- if PORT_ALL.append("Ethernet%d" % (port_idx * 4)) %}{%- endif %}
+    {%- endfor %}
+{%- endmacro %}
+
+{%- macro generate_buffer_pool_and_profiles() %}
+    "BUFFER_POOL": {
+        "ingress_lossless_pool": {
+            "size": "{{ ingress_lossless_pool_size }}",
+            "type": "ingress",
+            "mode": "dynamic"
+        },
+        "ingress_lossy_pool": {
+            "size": "{{ ingress_lossy_pool_size }}",
+            "type": "ingress",
+            "mode": "dynamic"
+        },
+        "egress_lossless_pool": {
+            "size": "{{ egress_lossless_pool_size }}",
+            "type": "egress",
+            "mode": "dynamic"
+        },
+        "egress_lossy_pool": {
+            "size": "{{ egress_lossy_pool_size }}",
+            "type": "egress",
+            "mode": "dynamic"
+        }
+    },
+    "BUFFER_PROFILE": {
+        "ingress_lossless_profile": {
```

Comment:
@akokhan You could use the arista platform as a reference. As these are the changes that MSFT approved [here](https://github.com/Azure/sonic-buildimage/pull/2972/files). Specifically, you should be good to remove the ingress lossless profile here, as it will be generated based on the pg_profile_lookup.ini file